src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import mate.academy.spring.validation.annotation.Email;
+import mate.academy.spring.validation.annotation.Password;
+import mate.academy.spring.validation.annotation.ValidPassword;
+
+@ValidPassword
 public class UserRequestDto {
+    @Email
     private String email;
+    @Password
     private String password;
+    @Password
+    private String repeatPassword;
```

Comment:
I don't think there is reason to check repeatPassword format
It's enough to check password and than to check if repeatPassword equals password